app/templates/events/view.hbs

```diff
@@ -1,6 +1,74 @@
 {{#unless (includes session.currentRouteName 'events.view.edit')}}
-  {{!-- Tabs for the event view page will come here --}}
-  {{outlet}}
+  <div class="ui grid stackable">
+    <div class="row">
+      <div class="four wide column">
+        <h2 class="ui header">Fossasia Test</h2>
+      </div>
+      <div class="twelve wide column {{unless device.isMobile 'right aligned'}}">
+        {{#if device.isMobile}}
+          <button class="ui button"><i class="unhide icon"></i></button>
+          <div class="ui icon buttons">
+            <button class="ui button "><i class="check icon"></i></button>
+            <button class="ui button "><i class="copy icon"></i></button>
+          </div>
+          <button class="ui red button icon"><i class="trash icon"></i></button>
+        {{else}}
+            <button class="ui button labeled icon small">
+              <i class="unhide icon"></i>
+              {{t 'Preview'}}
+            </button>
+            <div class="ui small labeled icon buttons">
+              <button class="ui button ">
+                <i class="check icon"></i>
+                {{t 'Publish'}}
+              </button>
+              <button class="ui button ">
+                <i class="copy icon"></i>
+                {{t 'Copy'}}
+              </button>
+            </div>
+            <button class="ui red button labeled icon small">
+              <i class="trash icon"></i>
+              {{t 'Delete'}}
+            </button>
+        {{/if}}
+      </div>
+    </div>
+  </div>
+
+  <div class="row" style="padding-top: 1em">
+    <div class="sixteen wide column">
+      {{#if device.isMobile}}
+        {{#ui-dropdown class='vertical selection'}}
+          <i class="dropdown icon"></i>
+          <div class="default text">{{t 'Overview'}}</div>
+          <div class="menu">
+            <div class="item" data-value="overview">{{t 'Overview'}}</div>
+            <div class="item" data-value="tickets">{{t 'Tickets'}}</div>
+            <div class="item" data-value="schedular">{{t 'Schedular'}}</div>
+            <div class="item" data-value="sessions">{{t 'Sessions'}}</div>
+            <div class="item" data-value="speakers">{{t 'Speakers'}}</div>
+            <div class="item" data-value="export">{{t 'Export'}}</div>
+          </div>
+        {{/ui-dropdown}}
+      {{else}}
+          <div class="ui fluid pointing secondary menu">
```

Comment:
add a `stackable` class to it. and remove `secondary` class on mobile. See the result. (and it's okay if the menu takes up space on top for now). We'll handle all tabs together in a better way later.